### ISSUE TITLE
Introduce an Accessible Theme

### ISSUE THREAD

**Issue by brianpierson2020** [navidrome/navidrome#259]
It would be great to introduce an Accessible Theme. Maybe I will work on it later, just dropping the idea here for the backlog.

**Comment by jvoisin** [comment]
What is an "Accessible theme"?

**Comment by brianpierson2020** [comment]
Accessibility theme, high contrast for people with vision problems.

**Comment by jvoisin** [comment]
What would you change to the *light* theme to make it more accessible?

**Comment by brianpierson2020** [comment]
I noticed that you disagreed and that if fine, everyone is entitled to their opinion so I don't expect you to support it. Feel free to disagree with the idea, as I mentioned I just dropped the idea. So maybe someday we can develop an additional theme that is more friendly to people with disabilities. Nowadays many GUI's that are themeable have a high contrast theme probably including your own OS, mobile devices, IDE. My point is: it would be a nice addition to have someday, not the main priority or goal of course.

If you do want to learn more about the matter, there is actually quite some documentation of the related subject for example [WCAG 2.0 AA](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines) as developed by the  [WAI](https://www.w3.org/WAI/).

**Comment by jvoisin** [comment]
I don't disagree, I'm just curious about how you think the light theme doesn't have enough contrasts.
